tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Differ;
+
+use PhpCsFixer\Differ\SebastianBergmannDiffer;
+use PHPUnit\Framework;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Differ\SebastianBergmannDiffer
+ */
+final class SebastianBergmannDifferTest extends Framework\TestCase
+{
+    public function testIsDiffer()
+    {
+        $differ = new SebastianBergmannDiffer();
+
+        $this->assertInstanceOf('PhpCsFixer\Differ\DifferInterface', $differ);
+    }
+
+    public function testDiffReturnsDiff()
+    {
+        $old = 'foo';
+        $new = 'bar';
+
+        $diff = <<<'TXT'
+--- Original
++++ New
+@@ @@
+-foo
++bar
```

Comment:
I think we shall have more complicated diffs here as well. this one is really trivial